src/CMakeLists.txt

```
@@ -48,6 +48,7 @@ if (MSVC)
         /volatile:iso
         /Zc:externConstexpr
         /Zc:inline
+        /Zc:preprocessor # Use standards-conforming preprocessor
```

Place the comment above add_compile_options

```suggestion
        /Zc:preprocessor
```

---

Sorry, Reading: 100. Give me a minute